README.md

```diff
@@ -1 +1,57 @@
+# ruby-selenium-browserstack
 
+## Prerequisite
+a. Ruby setup on your machine
+```
+ruby -v
+```
+
+b. bundler gem
+```
+gem install bundler
+```
+
+## Steps for running test session
+
+1. Clone the repo, navigate to the cloned folder then install the dependencies:
+```
+bundle install
+```
+2. Configure the capabilties for the test and use your credentials <br>
+(Context: For running single test session). <br>
+i. Navigate to ./scripts/single.rb <br>
+ii. Change the capabilities and swap the credentials.
+  ```ruby
+# change capabilities
+caps = Selenium::WebDriver::Remote::Capabilities.new
+caps['device'] = 'iPhone 11'
+caps['realMobile'] = 'true'
+caps['os_version'] = '14.0'
+caps['javascriptEnabled'] = 'true'
+caps['name'] = 'BStack-[Ruby] Sample Test' # test name
+caps['build'] = 'BStack Build Number 1' # CI/CD job or build name
+driver = Selenium::WebDriver.for(:remote,
+  :url =>  "https://USER_NAME:ACCESS_KEY@hub-cloud.browserstack.com/wd/hub", # Use your browserstack username and accesskey
```

Comment:
can you make it configurable via env variable?

Reply:
Sure